Misc/ACKS

```diff
@@ -346,6 +346,7 @@ David M. Cooke
 Jason R. Coombs
 Garrett Cooper
 Greg Copeland
+Max Coplan 🤘
```

Comment:
```suggestion
Max Coplan
```

I appreciate your enthusiasm but let's keep the name consistent with the rest of the file ;)